co-simulation/fed/mosaic-infrastructure/src/main/java/org/eclipse/mosaic/fed/infrastructure/ambassador/InfrastructureInstance.java

```diff
@@ -35,40 +37,48 @@ public class InfrastructureInstance {
     private InetAddress targetAddress;
     private int rxMessagePort;
     private int timeSyncPort;
+    private int simulatedInteractionPort;
```

Comment:
I think that is probably overkill, especially since all the getter/setter methods have documentation. I can add documentation to the constructor as well but I do not think it is necessary for every class member

Reply:
I understand, in my opinion when doing these reviews, we should judge the documentation based on whether it is reasonable to expect someone foreign to the code to be able to read it and understand it without requiring an unreasonable amount of investigation. With that thought in mind I think documenting your methods declarations is a must/especially any public methods that others may use. Additionally, a combination of variable naming and inline comments should be able to give meaning to your variables. I do not think it is important to add a comment to a location variable explicitly saying it is the location, but if you have 5 location variables i think it is important to document inline or constructor what the difference between the 5 location variables are. In that spirit I could add documentation to the port members but I think in this case they are named well and have constructor documentation that makes this not completely necessary.